### PR TITLE
Remove celo documentation from core

### PR DESCRIPTION
**Pull request by mpetrun5**
Remove celo documentation from core to celo module

## Related Issue Or Context

Closes: #188 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Documentation

## Checklist:

- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have ensured that all acceptance criteria (or expected behavior) from issue are met
- [x] I have updated the documentation locally and in chainbridge-docs.
- [ ] I have added tests to cover my changes.
- [ ] I have ensured that all the checks are passing and green, I've signed the CLA bot

### CODE REVIEW

**Review by waymobetta — COMMENTED** [review]
All looks fine but will wait until we discuss before submitting final approval.

I guess we can put CLI examples in module readmes then link back to these within core readme, that way management of cmds is held within each respective module but also there is visibility about the modules/cmds that core **can** support if modified sufficiently?

**Comment by P1sar** [comment]
> 

I guess modules should have links for CORE cli since it is the same, just un case that module have some specific  CLI it should be described in modules README.
@mpetrun5 Could you please move to celo-module repo all CELO related reamde (except CLI)

**Review by waymobetta — APPROVED** [review]

**Review by MakMuftic — APPROVED** [review]

**Pull request merged**
